eng/Version.Details.xml

```diff
@@ -1,33 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Razor.Symbols.Transport" Version="7.0.0-preview.2.22081.4">
+    <Dependency Name="Microsoft.AspNetCore.Razor.Symbols.Transport" Version="7.0.0-preview.2.22115.3">
       <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>427991ec87d4df83fdb7a24425c3cce550cf2423</Sha>
+      <Sha>ca1728b83851de4211477ab37a2460bf567544d3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.Razor.Tooling.Internal" Version="7.0.0-preview.2.22081.4">
+    <Dependency Name="Microsoft.CodeAnalysis.Razor.Tooling.Internal" Version="7.0.0-preview.2.22115.3">
       <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>427991ec87d4df83fdb7a24425c3cce550cf2423</Sha>
+      <Sha>ca1728b83851de4211477ab37a2460bf567544d3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Tooling.Internal" Version="7.0.0-preview.2.22081.4">
+    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Tooling.Internal" Version="7.0.0-preview.2.22115.3">
       <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>427991ec87d4df83fdb7a24425c3cce550cf2423</Sha>
+      <Sha>ca1728b83851de4211477ab37a2460bf567544d3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.Internal.Transport" Version="7.0.0-preview.2.22081.4">
+    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.Internal.Transport" Version="7.0.0-preview.2.22115.3">
       <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>427991ec87d4df83fdb7a24425c3cce550cf2423</Sha>
+      <Sha>ca1728b83851de4211477ab37a2460bf567544d3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X.Internal.Transport" Version="7.0.0-preview.2.22081.4">
+    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X.Internal.Transport" Version="7.0.0-preview.2.22115.3">
       <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>427991ec87d4df83fdb7a24425c3cce550cf2423</Sha>
+      <Sha>ca1728b83851de4211477ab37a2460bf567544d3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Sdk.Razor" Version="6.0.0-alpha.1.21072.5">
-      <Uri>https://github.com/dotnet/aspnetcore</Uri>
-      <Sha>20b6779cf3af2fed6a8fe64a0865cfab50b776bb</Sha>
+    <Dependency Name="Microsoft.NET.Sdk.Razor" Version="7.0.100-preview.2.22115.8">
```

Comment:
Can we remove this entirely? We shouldn't add a dependency from sdk -> razor.tooling.

Reply:
Is it important that the repo take the runtime version that the razor sdk depends on? That's the question to answer. If not, then just have a separate runtime subscription is fine.

Reply:
> I don't know if you need one. Maybe we can get help from @dougbu / @wtgodbe with this?

I am unsure how razor-compiler uses the Razor SDK but it feels odd to require the latest NETCore.App in a tooling repo, especially if we're trying to stay up-to-date w/ the latest SDK.

Having just one runtime dependency in this file also seems odd given the manual versioning of System.Text.Encodings.Web and similar packages. Suggest we choose one approach (manual or automated) for all dependencies from that repo. This is secondary to deciding whether we need to revise both the SDK and the NETCore runtime automatically.

Another option might be to subscribe to dotnet/sdk for the latest .NET SDK and tease that bundle apart for any bits in the Razor part of it needed here. That kind of brings everything together in one dependency.

One question to end my ramblings: Are loops e.g. razor-compiler ➡️ sdk ➡️ razor-compiler ... going to be a problem❔ I know dotnet/arcade does something in an even tighter loop (depending on itself) but wonder if there are gotchas here.

Reply:
> Is it important that the repo take the runtime version that the razor sdk depends on?

> I am unsure how razor-compiler uses the Razor SDK but it feels odd to require the latest NETCore.App in a tooling repo, especially if we're trying to stay up-to-date w/ the latest SDK.

@NTaylorMullen?

Reply:
>  Is it important that the repo take the runtime version that the razor sdk depends on?

Most (probably all) of the razor-compiler packages are ns2.0 targeting since they need to run in VS. So the .NET Core runtime isn't really in the picture.

Reply:
Tooling typically absorbs newer changes than SDK

Reply:
If I followed this discussion correctly, the appropriate change would be: https://github.com/dotnet/razor-tooling/pull/6088/commits/d0d0e0218a5ef38bd0905da4316e2febd93b9d0f

Please let me know if that isn't the case.